src/test/java/app/daos/impl/GameDAOImplTest.java

```diff
@@ -0,0 +1,121 @@
+package app.daos.impl;
+
+import app.Populator;
+import app.config.HibernateConfig;
+import app.entities.Game;
+import app.entities.Genre;
+import app.entities.Platform;
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.EntityNotFoundException;
+import org.junit.jupiter.api.*;
+
+import java.time.LocalDate;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameDAOImplTest {
+
+    private static Populator populator;
+    private static GameDAOImpl gameDAO;
+    private List<Genre> genres;
```

Comment:
Perhaps add an empty line between private static variables and private class variables to improve readability